CMakeLists.txt

```diff
@@ -69,3 +81,30 @@ include(${BASE_LOCATION}/toolchain/post-build.cmake)
 
 # Add JLink commands
 include(${BASE_LOCATION}/toolchain/jlink.cmake)
+
```

Comment:
hey this seems useful, do you think it could be moved info a function exposed by efm32-base to simplify this file (and turning it on and off)?

Reply:
:+1: